### PR TITLE
Tailwind number field

### PR DESCRIPTION
**Pull request by OsamaAbdellateef**

### CODE REVIEW

**Review by OsamaAbdellateef — COMMENTED** [review]
PR is reviewed

File: packages/components/src/NumberField/NumberField.test.tsx [hunk 1]
```diff
@@ -1,121 +1,32 @@
...
-          },
-
-          '&:error-group': {
-            borderColor: 'red',
-          },
-        },
-      },
+      group: cva(),
+      label: cva(),
+      helpText: cva(),
```
Comment (sarahgm) [inline packages/components/src/NumberField/NumberField.test.tsx:17]: why you don't set any classNames?

File: packages/components/src/NumberField/NumberField.tsx [hunk 2]
```diff
@@ -130,26 +117,15 @@ export const NumberField = forwardRef<HTMLInputElement, NumberFieldProps>(
...
           {...mergeProps(groupProps, focusProps, hoverProps)}
           {...stateProps}
         >
           {showStepper && (
-            <StepButton
-              direction="down"
-              css={styles.stepper}
-              {...decrementButtonProps}
-            />
+            <StepButton direction="down" {...decrementButtonProps} />
```
Comment (sarahgm) [inline packages/components/src/NumberField/NumberField.tsx:128]: why no `className.stepper` ?

File: packages/components/src/NumberField/NumberField.tsx [hunk 3]
```diff
@@ -163,13 +139,9 @@ export const NumberField = forwardRef<HTMLInputElement, NumberFieldProps>(
             {...stateProps}
           />
           {showStepper && (
-            <StepButton
-              direction="up"
-              css={styles.stepper}
-              {...incrementButtonProps}
-            />
+            <StepButton direction="up" {...incrementButtonProps} />
```
Comment (sarahgm) [inline packages/components/src/NumberField/NumberField.tsx:142]: why no `className.stepper` ?

**Comment by sarahgm** [comment]
@OsamaAbdellateef  where are the styles file? :o

**Comment by OsamaAbdellateef** [comment]
> @OsamaAbdellateef where are the styles file? :o

Actually there's no styling for number filed in old theme-core , if you want me to just create file then this is fine :)

**Comment by sarahgm** [comment]
I think you have to even if its empty or am I wrong? @OsamaAbdellateef @sebald

**Comment by sebald** [comment]
There was none, but we need to make one 😄

**Comment by OsamaAbdellateef** [comment]
> There was none, but we need to make one smile

Added now :)

**Review by sarahgm — APPROVED** [review]

**Pull request merged**